### DISCUSSION TITLE
Feature Request: Auto Delete Browser History after X amount of time

### DISCUSSION THREAD

**Discussion by JoshuaIyalagha** [Floorp-Projects/Floorp#928] [Feature Request]
Hello, 
Great work on the Floorp Browser. I truly appreciate it. 
I would like to make a feature request: It would be nice if there is an option to autodelete browser history after a preset amount of time. My suggested presets are 1 week, 2 weeks, 1 month, 3 months as well as a custom [] days option where the user inputs the number of days they want.

**Comment by creeper-0910** [comment]
If you explain how the feature will help, it is more likely to be implemented.

**Reply by JoshuaIyalagha** [reply]
For privacy purposes I regularly delete my internet history every other month or so. I also purge my data on my google accounts. This would basically help me not have think about it while using Floorp. 

I am trying out different/non google services at the moment to not have to give google my data anymore. 

The auto delete history option is already available on Vivaldi. I personally do not think it is critical or high priority. I just hope you consider implementing it when it is convenient for you. Take care!

**Reply by Ryosuke-Asano** [reply]
Ok. IMO, this is great feature. I will consider about it
